### ISSUE TITLE
When people sign up, tell them how the act of uploading is interpreted

### ISSUE THREAD

**Issue by neilb** [andk/pause#251]
When people sign up for a PAUSE account, we should have a statement in there along the lines of

> All files uploaded to your author directory will be copied to more than 250 [CPAN mirror sites](https://www.cpan.org/SITES.html) around the world. The act of uploading a file is therefore interpreted as implicit permission to mirror, distribute, and process that file.

We should try and keep it concise (greater likelihood of people reading it), but we might consider including something along the lines of:

> There are no exceptions to this: you cannot specify that you don't want your uploads to Russia, China, or the USA (insert favourite oppressive regime), for example.

**Comment by charsbar** [comment]
I don't think this is a good idea because nobody wants their homeland to be named like this.

**Comment by neilb** [comment]
Yeah, sorry, the oppressive regime joke was for here (github), not suggesting that goes into PAUSE.

I'll edit my suggestion.

**Comment by karenetheridge** [comment]
Has this been an issue in the past? Has some author requested that their distribution not be mirrored to (for example) Libya, North Korea or Syria?

I don't know if there are specific anti-sanction laws in some jurisdictions that would prohibit open source works from being sent to specific nations, or even if there are any mirrors in countries people are concerned about (although new mirrors could always appear in the future, I suppose).

**Comment by neilb** [comment]
@karenetheridge this came out of discussions between PAUSE admins related to the PAUSE governance model.
